draft-ietf-quic-manageability.md

```diff
@@ -809,20 +816,83 @@ off the connection ID may therefore cause undetectable and unrecoverable loss
 of state in the middle of a connection. Use of connection ID specifically
 discouraged for NAT applications.
 
-## Passive Network Performance Measurement and Troubleshooting
+If a NAT hits an operational limit, it is recommended to rather drop the
+initial packets of a flow (see allso {{sec-filtering}}),
+which potentially triggers a fallback to TCP, than using the connection ID to
+muliplex mulitple connections on the same IP address port pair which risks
+breakage later on connection ID change.
+
+## Address Rewriting to Ensure Routing Stability
+
+If the routers or switches in the server infrastructure rely on address-port
+4-tuple, a NAT rebinding or any other address migration on the client side,
+even if that could be handled by the QUIC server, will cause packets to be
+delivered to the wrong server. In this case it seems temping to use NAT in
```

Comment:
This sentence is too long.

Reply:
I know. That's my German gens come through. I will break it up in two sentences.